phpstan.neon

```diff
@@ -8,6 +8,7 @@ parameters:
     inferPrivatePropertyTypeFromConstructor: true
     checkMissingIterableValueType: false
     checkGenericClassInNonGenericObjectType: false
+    reportUnmatchedIgnoredErrors: false
```

Comment:
Why do we want to add this? Isn't it better to just remove the un-matched errors?

Reply:
Because it fails the ci even if one fixes some errors, the un-matched errors can be removed at a later stage when updating the baseline, not every pr which fixes some error should be forced to also update the baseline.